<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="jaya-manesh" failures="1" tests="4" name="Test" time="0.038" errors="0" timestamp="2024-08-03T10:43:46 IST">
  <testcase classname="testng.RetryTest" name="demo" time="0.008">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase classname="testng.RetryTest" name="demo" time="0.001">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase classname="testng.RetryTest" name="demo" time="0.001">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase classname="testng.RetryTest" name="demo" time="0.001">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testng.RetryTest.demo(RetryTest.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- demo -->
</testsuite> <!-- Test -->
